lib/assert.js

```diff
@@ -281,16 +274,18 @@ assert.throws = function throws(block, error, message) {
       expected: error,
       operator: 'throws',
       message: `Missing expected exception${details}`,
-      stackStartFn: throws
+      stackStartFn: assert.throws
```

Comment:
Maybe name `assert.throws` as a top level function and reference it directly here to avoid being messed up by monkey-patched `assert.throws`

Reply:
👌